pkg/sql/importer/read_import_avro.go

```diff
@@ -77,6 +96,12 @@ func nativeToDatum(
 		d = tree.NewDFloat(tree.DFloat(v))
 	case float64:
 		d = tree.NewDFloat(tree.DFloat(v))
+	case time.Time:
+		return mapDateTime(x, targetT)
+	case time.Duration:
+		// goAvro returns avro cols of logical type time as time.duration
+		dU := x.(time.Duration) / time.Microsecond
```

Comment:
[nit] Doesn't `v` already contain the converted type?

```suggestion
		dU := v / time.Microsecond
```

Similarly in the case above.